src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -1005,7 +1007,7 @@ tr.status-warning td {
     background: var(--header-background);
     border-radius: 6px;
```

Comment:
should this use `var(--summary-status-border-width)` ?

Reply:
Yes! Fixed. Thanks.